M/MAGEMin/build_tarballs.jl

```diff
@@ -31,9 +30,13 @@ make -j${nproc} CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" lib
 make -j${nproc} CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" all
 
 install -Dvm 755 libMAGEMin.dylib "${libdir}/libMAGEMin.${dlext}"
-install -vm 644 src/*.h "${includedir}"
 install -Dvm 755 MAGEMin* "${bindir}/MAGEMin${exeext}"
 
+# store files
+install -vm 644 src/* "${includedir}"
+#install -vm 644 src/* "${sourcedir}"     
```

Comment:
I'm not sure what was the purpose of this line, there is no `${sourcedir}` environment variable.
```suggestion
```

Reply:
yes, removed that now